src/main.rs

```
@@ -1,3 +1,28 @@
+use std::collections::HashMap;
+
+const DB_FILENAME: &str = "kv.db";
+
 fn main() {
-    println!("Hello, world!");
+    let database = Database::new().expect("Failed to create database!");
```

clippy

unused variable

(warning)